src/main/java/pl/edu/mimuw/logic/Variable.java

```diff
@@ -0,0 +1,54 @@
+package pl.edu.mimuw.logic;
+
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+
+public class Variable implements Formula {
+
+  String varName;
```

Comment:
ta widoczność mnie boli 😞

Reply:
teraz jak patrze to mnie też, cóż trzeba było lepiej się przyjrzeć